docusaurus.config.js

```diff
@@ -283,20 +283,33 @@ const config = {
               "/developers/deploy-smart-contracts",
             ],
           },
+          {
+            to: "/zero-knowledge-glossary",
+            from: "/reference/glossary",
+          },
           {
             to: "/build-on-linea/use-linea-testnet",
             from: [
               "/get-started",
-              "/get-started/bridge-funds",
               "/get-started/fund",
               "/get-started/configure-metamask",
               "/get-started/quickstart",
               "/use-zkevm",
-              "/use-linea/fund",
-              "/use-linea/bridge-funds",
               "/use-linea/index.md",
             ],
           },
+          {
+            to: "/build-on-linea/use-linea-testnet/bridge-funds",
```

Comment:
Is this a duplicate of line 310?

Reply:
did you mean to use `get-started`? and then collapse that into one entry with two from links in teh list

Reply:
oh yeah i guess it is a duplicate. i am not sure what get-started section is supposed to link to. oliver made those initial redirects so i didn't want to change what i wasn't sure about. ill get rid of the duplicate one though